pkg/distro/fedora/distro.go

```diff
@@ -454,6 +454,36 @@ var (
 		basePartitionTables:    minimalrawPartitionTables,
 		requiredPartitionSizes: requiredDirectorySizes,
 	}
+
+	minimalrawZstdImgType = imageType{
+		name:        "minimal-raw-zst",
```

Comment:
(wondering) should this be `minimal-raw-zstd` (notice the leading `d`) ? I know the extension if `zst` (which I personally find very confusing) so maybe fine? idk

Reply:
I indeed picked it based on the extension so the image type is related to the output artifact, I did things in the same style in #1233.